packages/phpstan-extensions/src/Testing/AbstractServiceAwareRuleTestCase.php

```diff
@@ -18,6 +19,14 @@ abstract class AbstractServiceAwareRuleTestCase extends RuleTestCase
      */
     private static $containersByConfig = [];
 
+    public function analyse(array $filePaths, array $expectedErrorsWithLines): void
+    {
+        try {
+            parent::analyse($filePaths, $expectedErrorsWithLines);
+        } catch (Throwable $throwable) {
+        }
```

Comment:
This should be solved if possible, I'll look into it

Reply:
@samsonasik Next time please refer the issue in conde comment, so we know why this magic is here.